README.md

```diff
@@ -62,28 +62,34 @@ The easiest way to compile the code and run the system locally is using [Docker]
 
 Don't forget to run the docker daemon!
 
-## Pull a pre-built image
-We publish new docker images for all commits to `trunk`.
-You can find the images [in the Github Container Registry](https://github.com/mit-dci/opencbdc-tx/pkgs/container/opencbdc-tx).
-
-
 ## Launch the System
 
 **Note:** You will need to both run the system and interact with it; you can either use two shells, or you can add the `--detach` flag when launching the system (note that it will then remain running till you stop it, e.g., with `docker stop`).
 Additionally, you can start the atomizer architecture by passing `--file docker-compose-atomizer.yml` instead.
 
-_The below configuration uses the pre-built image from Github Container Registry.
-To run and test your changes to the codebase you can follow the [Development section](#development)._
+_The commands below will build a new image every time that you run it.
+You can remove the `--build` flag after the image has been built to avoid rebuilding.
+To run the system with our pre-built image proceed to the [next section](#launch-the-system-with-a-pre-built-image) for the commands to run._
 
 1. Run the System
    ```terminal
-   # docker compose --file docker-compose-2pc.yml up
+   # docker compose --file docker-compose-2pc.yml up --build
```

Comment:
This is just removing a step in the process by using the flag instead of a separate run of `docker build`